packages/swirl-components/src/components/swirl-image-grid-item/swirl-image-grid-item.tsx

```diff
@@ -132,6 +196,32 @@ export class SwirlImageGridItem {
           ) : (
             <div class="image-grid-item__loading-placeholder"></div>
           )}
+
+          {this.showGifControls && (
+            <swirl-stack
+              class="image-grid-item__gif-controls"
+              orientation="horizontal"
+              spacing="4"
+            >
+              <div
+                class="image-grid-item__gif-controls__icon image-grid-item__gif-controls__icon--button"
+                onClick={this.gifPaused ? this.playGif : this.pauseGif}
+                role="button"
```

Comment:
Issue: If you don't use a real `<button>` you need to implement keyboard controls and make the element focusable. Could you use a real button here instead?

Reply:
changed it to a button. (i wish buttons didn't have innate stylings attached)

Reply:
Trick: `all: unset;`

Reply:
`type="button"` to prevent form submission would be nice as well.

Reply:
added back the button type